modules/ROOT/pages/starting-and-stopping-mule-esb.adoc

```diff
@@ -0,0 +1,180 @@
+:keywords: mule, esb, deploy, launch, run, terminate
+
+[NOTE]
+====
+This section deals with deploying to standalone instances of the Mule Runtime. To take full advantage of managing and monitoring features, you can deploy to the same Mule Runtime via the xref:runtime-manager::index.adoc[Runtime Manager]. See xref:runtime-manager::deployment-strategies.adoc[Deployment Strategies] for a deeper look at the different deployment alternatives offered by the platform.
+====
+
+Mule uses the Java Service Wrapper to control the Java Virtual Machine (JVM) from your native OS. The wrapper provides many options and features, including the ability to run Mule as a Unix daemon or install or remove Mule as a Windows Service. The wrapper can handle system signals and start parameters, and overall provides much better interaction between the JVM and the underlying OS.
+
+
+== Startup and Shutdown Script
+
+The wrapper is called by a script in `$MULE_HOME/bin`.
+
+The table below lists all the parameters that the script accepts.
+
+[%header,cols="30a,70a"]
+|===
+|Parameter |Description
+| `start` |Starts the Mule server in the terminal background.
+| `stop` |Stops the Mule server. The stopping process stops inbound endpoints to process in-flight messages. However due to a timeout duration not all messages are ensured to be processed before final shutdown occurs.
+| `restart` |Restarts the Mule server.
+| `status` | *(Linux/Unix only.)* Displays the status of the Mule server (`running` or `not running`) and the process PID if running.
+| `dump` | *(Linux/Unix only.)* Dumps the Mule wrapper's core to `$MULE_HOME/log/mule_ee.log`.
+| `console` |Start Mule in the terminal foreground (console mode). Same as running `mule` with no parameters.
+| `install` | *(Windows only.)* Install Mule as a Windows service.
+| `remove` | *(Windows only.)* Remove Mule from your Windows services.
+|===
+
+== Running Mule
+
+You can test if Mule runs in your system without errors by running the following commands:
+
+* On Windows environments:
++
+[source,powershell]
+----
+$ $MULE_HOME\bin\mule.bat
+----
+
+* On Linux/Unix environments:
++
+[source,console]
+----
+$ $MULE_HOME/bin/mule
+----
+
+These commands run Mule in foreground mode, and the startup script displays information on the terminal’s standard output. You can’t issue further commands on the terminal as long as Mule is running.
+
+To stop Mule, press `CTRL-C` in the terminal in which the script is running.
+
+== Starting Mule as a Windows Service
+
+To run Mule as a Windows service, you need to install it first by running:
+
+[source,powershell]
+----
+$ $MULE_HOME\bin\mule.bat install
+----
+
+Once installed, you can run Mule as a service:
+
+[source,powershell]
+----
+$ $MULE_HOME\bin\mule.bat start
+----
+
+[NOTE]
+====
+When Windows restarts, the Mule service stops the same way as using the `mule stop` command.
+The only situation where this could be different is if Windows kills the process due to a timeout.
```

Comment:
What is different if that happens? Suggest "If Windows kills the process due to a timeout, _____"